app/views/layouts/application.html.erb

```diff
@@ -1,21 +1,26 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js" lang="en">
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Chef Identity Service</title>
-  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= stylesheet_link_tag "application", media: "all" %>
+  <%= javascript_include_tag '//cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js' %>
```

Comment:
You can leave this for now, but I opened https://github.com/chef/chef-web-core/issues/52 to consider fixing it in the future.